resteasy-spring-boot-starter/pom.xml

```diff
@@ -37,18 +37,12 @@
             <organizationUrl>http://www.redhat.com</organizationUrl>
         </developer>
     </developers>
+
     <scm>
         <connection>scm:git:git@github.com:resteasy/resteasy-spring-boot.git</connection>
         <developerConnection>scm:git:git@github.com:resteasy/resteasy-spring-boot.git</developerConnection>
         <url>git@github.com:resteasy/resteasy-spring-boot.git</url>
     </scm>
-    <distributionManagement>
-        <repository>
-            <id>jboss-releases-repository</id>
-            <name>JBoss Releases Repository</name>
-            <url>https://repository.jboss.org/nexus/service/local/staging/deploy/maven2/</url>
-        </repository>
-    </distributionManagement>
```

Comment:
Same as above, what's the reason?

Reply:
Should we scatter these repository settings around?

Can we remove all these internal repository settings from the projects and make resteasy-spring-boot should consume the released RESTEasy version?

Reply:
@liweinan the reason why you're finding this in multiple poms is that they explicitly have the jboss-parent as parent. I don't think there's anything to refactor here, really, but if you think there is please consider doing that as part of a dedicated PR / jira.